src/ipc/UdsOp.cc

```diff
@@ -21,12 +21,12 @@ Ipc::UdsOp::UdsOp(const String& pathAddr):
     address(PathToAddress(pathAddr)),
     options(COMM_NONBLOCKING)
 {
-    debugs(54, 5, HERE << '[' << this << "] pathAddr=" << pathAddr);
+    debugs(54, 5, '[' << this << "] pathAddr=" << pathAddr);
```

Comment:
Agreed, but this PR automatically removes HERE, and nothing else should be required.